### PR TITLE
DDF-04395 Fixes fraction displayed when uploading

### PR DESCRIPTION
**Pull request by rececoffin**
#### What does this PR do?
Fixes the fractions that would sometimes show up incorrectly when uploading files.
#### Who is reviewing it? 

@cantstoptheunk 
@bellcc 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@jlcsmith 

#### How should this be tested?
* First upload any number of files in the uploader (ie. 2 files)

* Hit the clear button

* Then upload any number of files again (ie. 1 file)

* Hit the start button

* Ensure that the fraction displayed when uploading never exceeds 100%

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4395 ____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by cantstoptheunk** [comment]
build now

**Comment by cantstoptheunk** [comment]
Hero successful:
Uploaded single files then cleared and then uploaded single file again and produced the correct fraction and uploaded correctly.
Uploaded batch of files then cleared and then uploaded batch of files again and produced the correct fraction and uploaded correctly.
Attempted the above uploads by drag and drop and click to upload options.

**Comment by rececoffin** [comment]
The clear button isn't available when a batch upload is in progress. So this change won't effect those uploads. As for why this works, what I found was that even with the new BatchUpload model, the function that incremented `complete` was still called because it had been queued before the clear button was clicked. The clear function makes sure those are removed when the new BatchUpload model is used.

File: ui/packages/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ module.exports = Marionette.LayoutView.extend({
     this.render()
     this.onBeforeShow()
     this.resetDropzone()
+    this.uploadBatchModel.clear()
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js]:
```
    resetDropzone() {
       this.dropzone.options.autoProcessQueue = false
       this.dropzone.removeAllFiles(true)
    }
```

Might be worth grouping this into the resetDropzone function for future developers to have clarity.  What I'm suggesting above would have the same effect as this line.

**Comment by rececoffin** [comment]
build now

**Comment by cantstoptheunk** [comment]
Hero successful:
Uploaded single files then cleared and then uploaded single file again and produced the correct fraction and uploaded correctly.
Uploaded batch of files then cleared and then uploaded batch of files again and produced the correct fraction and uploaded correctly.
Attempted the above uploads by drag and drop and click to upload options.

**Pull request merged**